autoload/vim_dadbod_completion.vim

```diff
@@ -182,10 +182,10 @@ endfunction
 function! s:generate_query(db, query_key, ...) abort
   let base_query = db#adapter#dispatch(a:db, 'interactive')
   let Query = s:cache[a:db].scheme[a:query_key]
-  if a:0 > 0
```

Comment:
This condition caused errors to be flashed at the user whenever certain `#job#run()`s had not finished. The revised condition allows the operation to silently fail while waiting for the jobs to finish.